apps/web/src/ee/translations/pages/TranslationGroupsPage.tsx

```diff
@@ -45,9 +47,12 @@ export const TranslationGroupsPage = () => {
   return (
     <PageContainer>
       <Group position="apart" mx={24} mt={24} mb={16} align="center">
-        <Title size={2} data-test-id="translation-title">
-          Translations
-        </Title>
+        <Flex align="center" gap="75">
+          <Title size={2} data-test-id="translation-title">
+            Translations
+          </Title>
+          <DocsButton />
```

Comment:
This one was not added during when I created the docs button but now when ee web is moved here this is easy and possible.